### PR TITLE
Add CRS to BG-Flood Model output

### PR DESCRIPTION
**Pull request by caseyli92**
DESCRIPTION OF PR:
* Add CRS to the BG-Flood Model output 
* Paramatise `output_dir` (bg-flood model output directory) in `bg_flood_model.py` for consistency

## Developer Checklist
- [x] Make code change
- [ ] Update tests
  - [ ] Update / create new tests
  - [ ] Ensure these tests have the expected behaviour
  - [ ] Test locally and ensure tests are passing
- [ ] Update documentation
  - [ ] Readme
  - [ ] Docstrings
  - [ ] Comments
  - [ ] Wiki

## Reviewer Checklist
- [ ] Check new code for code smells
- [ ] Check new tests
  - [ ] Ensure adequate coverage
  - [ ] Check for code smells within tests
- [ ] Check if documentation needs updating
  - [ ] Readme
  - [ ] Docstrings
  - [ ] Comments
  - [ ] Wiki

### CODE REVIEW

File: src/flood_model/bg_flood_model.py [hunk 1]
```diff
@@ -115,8 +124,45 @@ class BGDEM(Base):
...
+    output_dir : pathlib.Path
+        BG-Flood model output directory.
+    """
+    list_of_files = output_dir.glob("*.nc")
+    try:
+        latest_file = max(list_of_files, key=os.path.getctime)
+    except ValueError:
+        latest_file = None
+        log.error(f"Missing BG-Flood Model output in: {output_dir}")
+    return latest_file
```
[the remark this answers is not in the document]
Comment (LukeParky) [reply src/flood_model/bg_flood_model.py:142]: Either do this error checking to make it explicit that None is an expected result (and add it to the docstring)

OR

Allow the error the be thrown, if you think that the latest file should exist in any normal case.

**Review by LukeParky — CHANGES_REQUESTED** [review]
Looks good, just have minor changes to `find_latest_model_output` that I think would improve maintainability and readability

File: src/flood_model/bg_flood_model.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (caseyli92) [reply src/flood_model/bg_flood_model.py:142]: Fixed.

**Review by LukeParky — APPROVED** [review]
LGTM
Nice changes, this should make some extra changes for fixing the netCDFs easier in the future

**Pull request merged**